code/modules/clothing/shoes/colour.dm

```diff
@@ -65,21 +65,21 @@
 	greyscale_config_worn = /datum/greyscale_config/sneakers_orange_worn
 
 /obj/item/clothing/shoes/sneakers/orange/attack_self(mob/user)
-	if (src.chained)
-		src.chained = null
-		src.slowdown = SHOES_SLOWDOWN
+	if (chained)
+		chained = null
+		slowdown = SHOES_SLOWDOWN
 		new /obj/item/restraints/handcuffs( user.loc )
-		src.icon_state = ""
+		icon_state = initial(icon_state)
 	return
 
 /obj/item/clothing/shoes/sneakers/orange/attackby(obj/H, loc, params)
 	..()
 	// Note: not using istype here because we want to ignore all subtypes
 	if (H.type == /obj/item/restraints/handcuffs && !chained)
 		qdel(H)
```

Comment:
Out of scope, but I don't see why we just move the handcuffs into the shoes's contents, and then move them out again at decuffing.